packages/autocomplete/src/examples/multiselect.md

```diff
@@ -0,0 +1,304 @@
+The `MultiSelect` layout operates similarly to the `Autocomplete`, but must
+manage the interaction between the Input and it's collection of Tags.
+
+This examples includes an example of "copy-to-clipboard" functionality for tags.
+
+```jsx
+const { MenuView, Item, AddItem, Separator } = require('@zendeskgarden/react-menus');
+const { FauxInput, Input, TextGroup, Label } = require('@zendeskgarden/react-textfields');
+const { Tag, Close } = require('@zendeskgarden/react-tags');
+const { Anchor } = require('@zendeskgarden/react-buttons');
+const { KEY_CODES, FieldContainer } = require('@zendeskgarden/react-selection');
+
```

Comment:
Should we be concerned that the following styling section isn't represented by the CSS components?

Reply:
I would say "yes" for the long-term, but out of the 4 implementations that are currently looking to use this component, only 1 (ticket props) is going to be able to match this look exactly.

Definitely think we need to eventually have some standardization around this in the future.